planex/spec.py

```diff
@@ -256,6 +258,29 @@ def series(self):
             return queue.series()
 
 
+def _parse_name(name):
+    """
+    Parse [name] a string composed by a workd and an optional number into the
+    name and the number as an integer (0 if not present). E. g.
+
+    - Source -> 0
+    - Patch001 -> 1
+    - Test12 -> 12
+
+    Raises KeyError if [name] cannot be parsed
+    """
+    matcher = re.compile(r"\A[a-zA-Z]+(\d*)\Z")
+    match = matcher.match(name)
+
+    if match is None:
+        raise KeyError(name)
+
+    (idx, ) = match.groups()
+    if idx == 0:
```

Comment:
Shouldn't this be `if idx == ''`?

Reply:
Yes, broken when rebasing, well spotted!